### DISCUSSION TITLE
Issue: Merge branch commit "pages build and deployment" workflow stuck on queued

### DISCUSSION THREAD

**Discussion by BrandonGwee** [community/community#110633] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Here's my repo: `https://github.com/BrandonGwee/stuff`

I tried using Commit (Amend) for the first time, before undoing the commit and just "Commit and Push"ing it. But after some attempts at branch merging, I found that the first branch merge is stuck on the "pages build and deployment" workflow stuck on queued.

-All attempts to cancel the action fail
-New commits don't trigger new "pages build and deployment" actions

I would really appreciate some help with this, thanks in advance!

**Comment by AidenJamesYt** [comment]
github push is currently experiencing degraded performance, meaning it is down for some people. To stay updated on the status, [visit the GitHub Status Site](https://www.githubstatus.com/) for live status updates.

**Comment by BrandonGwee** [comment]
It turns out the cancel request that was causing all the other requests to fail finally came through and the page is working fine, thanks for the help nonetheless
